peekingduck/pipeline/nodes/model/yoloxv1/yolox_model.py

```diff
@@ -74,9 +74,8 @@ def detect_ids(self, ids: List[int]) -> None:
         if not isinstance(ids, list):
             raise TypeError("detect_ids has to be a list")
         if not ids:
-            self.logger.info("Detecting all available classes.")
+            self.logger.info("Detecting all YoloX classes")
```

Comment:
Consider stylizing YOLOX the same way as it's done in the [paper](https://arxiv.org/abs/2107.08430)

Reply:
Re-styled as recommended.